readme.md

```diff
@@ -82,20 +84,36 @@ require('@zeit/ncc')('/path/to/input', {
   v8cache: false, // default
   quiet: false, // default
   debugLog = false // default
-}).then(({ code, map, assets }) => {
-  console.log(code);
-  // Assets is an object of asset file names to { source, permissions, symlinks }
-  // expected relative to the output code (if any)
+}).then(({ files, symlinks }) => {
+  // files: an object of file names to { source, permissions }
+  // symlinks: an object of symlink mappings required for the build
+  // The main file is located at 'index.js':
+  files['index.js'].source
 })
 ```
 
+Multiple entry points can be built by providing an object to build. In this case, those files are avaiable as additional output files:
+
+```js
+require('@zeit/ncc')({
+  entry1: '/path/to/input1',
+  entry2: '/path/to/input2
+}).then(({ files, symlinks }) => {
+  // named entry points are available at their names:
+  files['entry1.js'].source
```

Comment:
This is confusing. The user specified `entry1` but it came back as `entry1.js`? I think the key should match.

Reply:
So we allow users to output filenames without extensions?

Note that Node.js under `--experimental-modules` won't execute files without extensions, although yes that will only apply further in future.

Reply:
I see. I forgot about `ncc run` which probably needs to turn this into a real file. Will webpack work if the object is `{ "/path/to/input1.js": "/path/to/input1.js" }`?

Reply:
We still use relative paths in the entry object for Webpack, it then resolves them to the output path, which for our virtual filesystem fs is `/entry1.js` anyway. But yes, Webpack supports this fine.

I'm open to switching them back to names, although it is nice that we support both `.js` and `.mjs` this way too.